Documentation/planning/arcade-powered-source-build/implementation-plan.md

```diff
@@ -23,9 +23,7 @@ To get each repo building with the new source-build 5.0 plan, [Arcade-Powered So
 | 2 | msbuild | [Ben Villalobos](https://github.com/BenVillalobos) | ✔️ | 4 | | | | | |
 | 2 | NuGet.Client | [Fernando Aguilar Reyes](https://github.com/dominoFire) | ✔️ | 8 | | | | | |
 | 2 | templating | [Jose Aguilar](https://github.com/donJoseLuis) | ✔️ | ✔️ | | | | | |
-| 3 | extensions | [John Luo](https://github.com/JunTaoLuo) | ⏱ | 1 | | | | | |
-| 3 | aspnetcore-tooling | [John Luo](https://github.com/JunTaoLuo) | ⏱ | 3 | | | | | |
-| 3 | aspnetcore | [John Luo](https://github.com/JunTaoLuo) | ⏱ | 11 | | | | | |
+| 3 | aspnetcore | [John Luo](https://github.com/JunTaoLuo) | ⏱ | 14 | | | | | |
```

Comment:
I assume that the patches to aspnetcore-tooling would move to aspnetcore. However, I'm not sure which patch extensions is referring to so I'm not sure whether that patch will apply to runtime or aspnetcore or both.

Reply:
The patch for extensions is to set the DefaultNetCoreTargetFramework, so I think that one will probably not apply.  I think setting aspnetcore to 14 is sufficient.